distributed/diagnostics/task_stream.py

```diff
@@ -126,9 +126,18 @@ def rectangles(msgs, workers=None, start_boundary=0):
             if type(color) is not str:
                 color = color(msg)
 
+            if action == "transfer":
+                data = msg["nbytes"]
+                formatted_duration = (
+                    f"{format_time(stop - start)} {format_bytes_to_string(data)}"
```

Comment:
I think we need to avoid f-strings as they weren't introduced until 3.6

Reply:
cleaned up